lib/dbservice/sessions/session.ex

```diff
@@ -37,6 +38,7 @@ defmodule Dbservice.Sessions.Session do
     many_to_many(:users, User, join_through: "user_session", on_replace: :delete)
     many_to_many(:group, Group, join_through: "group_session", on_replace: :delete)
     has_many(:session_schedule, SessionSchedule)
+    has_many(:session_occurrence, SessionOccurrence)
```

Comment:
why.. this will be too big. live classes have 365 session occurrences for example

Reply:
It's opposite. We load session details along with session occurrence. It will be used in gurukul